notes/stories/television/tv-reddwarf1988.st.txt

```diff
@@ -1412,7 +1412,7 @@ what if I met my double [The crew first encountered enlightened versions of them
 
 :: Minor Themes
 sentient computer [A fire compromised the Red Dwarf computer Holly's ability to communicate using coherent phrases.]
-choosing who lives and who dies [The crew briefly found themselves in this predicament: If Lister and Cat were eliminated, then Rimmer and Kryten could commandeer the Starbug to safety; otherwise they'd all perish in seven minutes time.]
+choosing who dies and who lives [The crew briefly found themselves in this predicament: If Lister and Cat were eliminated, then Rimmer and Kryten could commandeer the Starbug to safety; otherwise they'd all perish in seven minutes time.]
```

Comment:
Huh, I wonder why we choose this turn of our phrase. I'm sure there was a very clever reason.